packages/server/lib/browsers/utils.ts

```diff
@@ -258,17 +262,10 @@ const getBrowsers = async () => {
     version,
     path: '',
     majorVersion,
-    info: 'Electron is the default browser that comes with Cypress. This is the default browser that runs in headless mode. Selecting this browser is useful when debugging. The version number indicates the underlying Chromium version that Electron uses.',
```

Comment:
This is the subject of #21769 - info display has been broken since 10.0.0. I've just removed it in lieu of adding the `info` display back in.